### ISSUE TITLE
Package unavailable for R3.0.0

### ISSUE THREAD

**Issue by Srividyanathan2006** [yihui/knitr#1301]
Hi Yihui,

I am trying to install knitr. and this is the message I get. I am unsure if any libraries need to be installed for ubuntu for knitr to be installed. I did not get any error messages indicating this. I was able to install for R3.3.0, however some of the packages that I want are available only for the old version of R3.0.

install.packages('knitr', repos='http://cran.rstudio.org')
Warning in install.packages :
 package ‘knitr’ is not available (for R version 3.0.0)

I tried downloading the package and install it from local directory with repos=NULL option. That did not work either.

thank you,
Srividya

**Comment by yihui** [comment]
knitr requires R >= 3.1.0: https://github.com/yihui/knitr/blob/master/DESCRIPTION#L81 R 3.0.0 was released more than 3 years ago, and knitr no longer supports it.

**Comment by Srividyanathan2006** [comment]
Hi Yihui, 

I understand that it is not supported. But some of the packages I require are in old version unfortunately. Just for people like me, who does not know R much and trying to get an idea of it -

I was able to install knitr1.6 version. I manually downloaded and installed all dependencies and I chose the archives that are in 2014 . It seems to have worked for me. 

I did not realize that I was so close to finding it out. I was trying to install a later version of knitr installation (1.11) when I downloaded and installed it - I should have checked the R version dependency.

thanks,
Srividya
